pipeline/tests/asm.rs

```diff
@@ -24,7 +24,7 @@ fn slice_to_vec<T: FieldElement>(arr: &[i32]) -> Vec<T> {
 fn sqrt_asm() {
     let f = "asm/sqrt.asm";
     let i = [3];
-    regular_test(f, &i);
+    regular_test_without_babybear(f, &i);
```

Comment:
why `without_babybear` again?

Reply:
since I added the change in your other comment above to also make proofs with p3, some asm tests fail because they have lookups/permutations inside the machine which we don't support